### DISCUSSION TITLE
Why sometimes it creates a tiddlers/$__StoryList_1.tid

### DISCUSSION THREAD

**Discussion by linonetwo** [TiddlyWiki/TiddlyWiki5#7404] [Q&A]
<img width="1310" alt="截屏2023-04-16 11 20 06" src="https://user-images.githubusercontent.com/3746270/232264559-4f42d706-f69e-4f39-9924-bea590596099.png">

I already add `$__StoryList.tid` to the git ignore. Because the storylist tid is constantly change, will cause lots of auto git backup. (TidGi auto git sync every 30min)

But sometimes nodejs wiki will try to create tid with `_1` that escape the git ignore.

I need to prevent this happen.

**Comment by tavin** [comment]
I don't know but you can gitignore a wildcard.

**Reply by linonetwo** [reply]
will it be something like `tiddlers/$__StoryList*.tid`?

Will Wildcard work with $ ?

**Reply by linonetwo** [reply]
Yes, I found `tiddlers/$__StoryList*.tid` works

**Comment by saqimtiaz** [comment]
Usually this happens when writing to $__StoryList.tid fails, due to permissions or being locked by another process.

**Reply by linonetwo** [reply]
Okay, maybe in TidGi app, previous nodejs process didn't quit properly, when switching between wiki...
